src/garage/tf/envs/base.py

```diff
@@ -20,12 +20,6 @@ class TfEnv(GarageEnv):
         env (gym.Env): the env that will be wrapped
     """
 
-    def __init__(self, env=None, env_name=''):
```

Comment:
hmm if you are removing this, why not remove GarageEnv totally? It is a TODO item anyway.